src/player.py

```diff
@@ -6,19 +6,29 @@ def __init__(self, name, currentRoom, items=[]):
         self.currentRoom = currentRoom
         self.name = name
         self.items = items
-
+    def __str__(self):
+        return f"\nHello {self.name}! \n"
+    #move function takes in a direction string
     def move(self, direction):
-        if direction == "n":
+    # checks string for both long and short form
+        if direction == "north" or direction == "n":
             self.currentRoom = self.currentRoom.n_to
-        if direction == "s":
+        if direction == "south" or direction == "s":
             self.currentRoom = self.currentRoom.s_to
-        if direction == "e":
+        if direction == "east" or direction == "e":
             self.currentRoom = self.currentRoom.e_to
-        if direction == "w":
+        if direction == "west" or direction == "w":
             self.currentRoom = self.currentRoom.w_to
+    #pickup function takes in an action string
+    def pickup(self, action):
```

Comment:
do players get a warning if they try to pick up an item that's not present?

Reply:
Added prompts to avoid this. Currently, if player tries to pick up an item not present, nothing happens (loop just resets)